.github/scripts/push-pypi.sh

```diff
@@ -0,0 +1,6 @@
+python -m pip install pipenv
```

Comment:
after line 1, you may need to do the below sometimes to explicitly create the environment with python 3.8
pipenv --python 3.8

Reply:
checking

Reply:
added pip --python 3.12